pymatgen/io/cif.py

```diff
@@ -1149,7 +1149,10 @@ def get_matching_coord(
             all_species_noedit = all_species.copy()  # save copy before scaling in case of check_occu=False, used below
             for idx, species in enumerate(all_species):
                 total_occu = sum(species.values())
-                if 1 < total_occu <= self._occupancy_tolerance:
+                if check_occu and total_occu > 1 + self._occupancy_tolerance:
```

Comment:
I think the original design of `occupancy_tolerance` is easier to understand. Changing to `1 + occupancy_tolerance` can cause more confusion.
As we discussed in #3816 the original implementation of occupany_tolerance should be fine!

Reply:
Thanks for your comment!

To me, the valid value range being `[expected, expected + tolerance]` instead of `[expected, tolerance]` seems more consistent with other parts of the code.

But I might revert this change now that we're trying to minimize breaking changes, clarify this in the docstring should suffice.